### ISSUE TITLE
[Bug] Power Conversion Arithmetic Error When Charging Batteries in a Mekanism Induction Matrix

### ISSUE THREAD

**Issue by robokid89** [turbodiesel4598/NuclearCraft#618]
When you charge any of the Voltaic Piles or Lithium Ion Batteries by placing them in a Mekanism Induction Matrix, It will charge them at 1/1000 of the power needed.  I was able to create Infinite power by just by charging the batteries in the Induction Matrix and then discharging them placing them down and by the power back into the Induction Matrix.

Using:
Nuclearcraft 2.18l
Mekanism 9.8.3.390

**Comment by turbodiesel4598** [comment]
@robokid89 I am unable to recreate this, with neither an induction matrix nor any of Mekanism's energy cells. What sort of energy cables are you using? Have you modified any configs?

**Comment by robokid89** [comment]
Using ultimate universal cables, but thats not the part thats bugged.  When you form an Induction Matrix, then open the GUI of the Induction Matrix and stick the Nuclearcraft battery in the slot to charge  an item.  Thats the part that is bugged.

I stuck a Tier 6 Lithium Ion Battery in my Induction Matrix (Only had a Advanced Cell and a Basic Provider) and it charged the 2084 MRF Lithium Ion Battery and consumed only 2 MRF from the Induction Matrix.

If you place the Lithium Ion Battery down in the world and use ultimate universal cables to send power from the Lithium Ion Battery back to the Induction Matrix. It will send the full 2048 MRF worth of power stored in the Lithium Ion Battery (at the cables max transfer rate).

No changes have been made to Nuclearcraft config and these are the only changes made to the Mekanism config (they shouldn't be affecting this).

- Mekanism
    + Disabled chunk loading ability of anchors
    + Increased the Cost in Joules of using the Atomic Disassembler as a weapon from 2000 to 5000
    + Disabled spawning of baby skeletons
    + Increased the Energy per operation tick of the digital miner from 100 to 400
    + Increased the Base Joules cost for a teleportation from 1000 to 10000
    + Increased the Flat additional cost for inter-dimensional teleportation from 10000 to 100000

**Comment by turbodiesel4598** [comment]
@robokid89 Ah, now I have recreated it to some degree (it's not 1/1000 for me, but still off), but I have actually been able to recreate this with Mekanism's own energy cells - I gave the induction matrix a bit of energy from a basic energy cell, and filled an ultimate energy cell in the GUI with it, and I got a similar result. It turned the initial 800 kRF into 6.84 MRF. So I think this is a bug with the induction matrix rather than NC's batteries.
